api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/cognito/AmazonCognitoAuthenticationProperties.java

```diff
@@ -56,4 +60,30 @@
      */
     @RequiredProperty
     private String userPoolId;
+
+    /**
+     * Describes the list of attributes that are to be fetched from the Cognito user pool (name)
+     * and then renamed using mappedTo values before they are put into a principal.
+     * An example might be to fetch {@code custom:custom_attribute_name} from user pool
+     * and rename to {@code attribute_name}.
+     */
+    private List<CognitoMappedAttribute> mappedAttributes = new ArrayList<>(0);
+
+    @RequiresModule(name = "cas-server-support-aws-cognito-authentication")
+    @Getter
+    @Setter
+    @Accessors(chain = true)
+    public static class CognitoMappedAttribute implements Serializable {
+        private static final long serialVersionUID = -6390112161784758805L;
+
+        /**
+         * Attribute name.
+         */
+        private String name;
+
+        /**
+         * The name that should be used to rename {@link #name}.
```

Comment:
Since this more or less moot now, since the class should be removed, but in generally, one should never refer to field names by name in the javadoc. Use the getter/setter constructs.